airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -482,3 +484,58 @@ def get_mapped_task_instance_try_details(
         map_index=map_index,
         session=session,
     )
+
+
+@task_instances_router.put(
+    "/updateTaskInstancesState",
+    responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND, status.HTTP_400_BAD_REQUEST]),
+)
+def set_task_instances_state(
```

Comment:
For consistency, I think it should be transformed as for dag_run into `patch_dag_run`.

A PATCH that can take only a state for now, but it will be easier to extend. It should also allow PATCHING the note.

Reply:
Just noticed you already have a PR opened for that.